source/_data/plugins.yml

```diff
@@ -279,6 +279,11 @@
       link: https://github.com/abhinaba-ghosh/cypress-react-selector
       keywords: [react, commands, locator]
 
+    - name: cypress-layout-inspector
+      description: 🕵️‍♂️ Simple utility to provide layout testing functionality to Cypress via custom assertions.
```

Comment:
```suggestion
      description: A simple utility to provide layout testing functionality to Cypress via custom assertions.
```

Reply:
@msmps Thanks for submitting the PR! I just had one request to update the description to remove the emoji and keep it text only. Once you give the approval and commit the change, happy to merge this PR.